.github/workflows/check-tests.yml

```diff
@@ -57,9 +57,17 @@ jobs:
             */*/node_modules
           key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
 
-      - name: Test 🧪
+      - name: Prepare Tests
         working-directory: './packages/millicast-sdk'
-        run: |
-          npm install
-          npx puppeteer browsers install chrome
-          npm run test
+        run: npm ci
+
+      - name: Unit Testing 🧪
```

Comment:
We could separate Unit Test and e2e Test in different jobs which will be faster I guess